src/lib.rs

```diff
@@ -859,6 +867,7 @@ impl TestKit {
     /// Returns reference to validator with the given identifier.
     ///
     /// # Panics
+    ///
     /// - Panics if validator with the given id is absent in test network.
     pub fn validator(&self, id: ValidatorId) -> &TestNode {
```

Comment:
It would be a questionable improvement. Wrappers ("strong typedefs") such as `ValidatorId` are only needed for "type safety". Making conversions easier can be error prone.

Reply:
Makes sense; thanks for clarification.